### DISCUSSION TITLE
What are some good losses' values when fine tuning XTTS v2?

### DISCUSSION THREAD

**Discussion by Jorvan758** [coqui-ai/TTS#3775] [General Q&A]
I've recently came across [this tutorial](https://www.youtube.com/watch?v=ds5LLIt5OLM) and decide to give XTTS a try (but running code in colab, instead of relying on a third party app). I've managed to generate a dataset with my own voice, following the LJSpeech format, and got no major problems running the base recipe for XTTS v2. However, I got no clue on what could be considered a good value for either avg_loader_time, avg_loss_text_ce, avg_loss_mel_ce not avg_loss in this case (seems like it hasn't been really discussed yet). I think I checked most of the few issues and discussions where these losses were shared by other people, but wasn't able to formulate any meaningful conclusions.
Would you be kind enough to share some of your knowledge/experience with me, pls?

**Comment by Ghassen-Cherni** [comment]
<img width="1051" height="151" alt="image" src="https://github.com/user-attachments/assets/9afa25f6-8a81-4186-bee6-dddae0163ca4" />


This is my current losses, the voice is good, but it hallucinates (generates random voices from time to time) and fails in certain sentences. Overall it's good for an MVP, but I wouldn't use it in prod. 

Hope this serves as a good benchmark
